django/db/models/fields/__init__.py

```diff
@@ -843,18 +866,12 @@ def get_flatchoices(self, include_blank=True,
         first_choice = blank_choice if include_blank else []
         return first_choice + list(self.flatchoices)
 
-    def _get_val_from_obj(self, obj):
```

Comment:
If you can try to make logical commits with the tests passing after each one as in https://github.com/django/django/pull/3770, I've found that quite helpful as a reviewer.